css/team.css

```diff
@@ -15,10 +15,14 @@ Styles for an individual page
 Page Component Styles (icon grid, team member card)
 ===================================
 */
-.black-text h3, .black-text h4, .black-text p {
```

Comment:
please review these last changes made for accessibility. including changes to header sizes and adding header classes, adding alt tags, changing background colour as back up for background img to ensure text is readable.

Reply:
Taking a look at things now :)